ContainerFileNotice

```diff
@@ -58,7 +58,7 @@ packages subject to the licenses identified below:
 |-------------------------------------------------------------+--------------+----------------------------------------------------------------------------------------|
 | k8s.io/apimachinery, v0.28.3                                | Apache-2.0   | https://github.com/kubernetes/apimachinery/blob/v0.28.3/LICENSE                        |
 |-------------------------------------------------------------+--------------+----------------------------------------------------------------------------------------|
-| k8s.io/client-go, v0.28.1                                   | Apache-2.0   | https://github.com/kubernetes/client-go/blob/v0.28.1/LICENSE                           |
+| k8s.io/client-go, v0.28.1                                   | Apache-2.0   | https://github.com/kubernetes/client-go/blob/v0.28.3/LICENSE                           |
```

Comment:
Oops, forgot to update the version in the left column. Fixing it in https://github.com/Xilinx-CNS/kubernetes-onload/pull/73.